docs/source/Getting_Started.rst

```diff
@@ -44,6 +44,70 @@ the following objects:
 
 To see examples of how these methods are used, please consult the demos.
 
+
+Parameters
+==========
+
+CaImAn gives you access to a lot of parameters and lets you adapt the analysis to your data. Parameters are stored in
+the ``params`` object in a set of dictionaries, sorted by the part of the analysis they are used in:
+
+-  ``data``: General params describing the dataset like dimensions, decay time, filename and framerate
+-  ``init``: Parameters for component initialization like neuron size ``gSig``, patch size etc.
+-  ``motion``: motion correction parameters (max shift size, patch size etc.)
+-  ``online``: Parameters specific for the online OnACID algorithm
+-  ``quality``: Parameters for component evaluation (spatial correlation, SNR and CNN)
+-  ``spatial``: Parameters used in detection of spatial components
+-  ``temporal``: Parameters used in extraction of temporal components and deconvolution
+
+Of these parameters, most have a default value that usually does not have to be adjusted. However, some parameters are
+crucial to be adapted to the specific dataset for proper analysis performance:
+
+-  ``fnames``: List of paths to the file(s) to be analysed. Memmap and hdf5 result files will be saved in the same directory.
+-  ``fr``: Imaging frame rate in frames per second.
+-  ``decay_time``: Length of a typical transient in seconds. Default of ``0.4``. (half-decay time?)
```

Comment:
Here I am not 100% sure how you define decay_time. It is the complete length of a transient, or the half-decay time (which is usually reported for fluorescence markers)? Following [the discovery paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3777791/), a transient of GCaMP6f for 1 AP lasts more than 1s, GCaMP6s even longer. If decay time actually means half-decay time, this should be addressed somewhere, otherwise the default of 0.4s might be too short for most markers.